search-enterprise/latest/using/filtering.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Filtering Paging and Sorting
+---
+
```

Comment:
I have sometimes been told to have some text between H1 and H2

Reply:
Will take care of it in the parent PR

Reply:
This will be tackled in the parent PR as I need to merge it ASAP.